build-system/conformance-config.textproto

```diff
@@ -4,3 +4,21 @@ requirement: {
   java_class: 'com.google.javascript.jscomp.ConformanceRules$BanThrowOfNonErrorTypes'
   error_message: 'Should not throw a non-Error object.'
 }
+
+requirement: {
+  type: BANNED_PROPERTY_CALL
+  error_message: 'Array.prototype.find is not allowed'
+  value: 'Array.prototype.find'
```

Comment:
since theres no actual source code change it isn't running `gulp check-types` LOL